code-challenges.test.js

```diff
@@ -21,9 +23,60 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// PSEUDO /////////////
+
+// The following paragraph is the blueprint to run the tests
+
+// describe("name of function",() => {
+//     it ("description of functionanlity what iy does", () => {
+//         expect ("invoke function").toEqual("expected result")
+//     })
+// }
+
+
+
+describe("fib", () => {
+    it("takes a number greater than 2 and returns an array", () => {
+        expect(fib(6)).toEqual([1, 1, 2, 3, 5, 8])
+    })
+})
+
+describe("fib", () => {
+    it("takes a number greater than 2 and returns an array", () => {
+        expect(fib(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+})
+
+
 
 // b) Create the function that makes the test pass.
 
+// PSEUDO // :
+
+// need a function with a loop that will iterate no more times than the value given
+// need to declare a variable that will hold the first 2 numbers
+// need to declare a variable that will hold the starting indexes of the array
+//need a loop that will initialize the iteration of the array values
+//use a for loop to iterate through the array
+//Within the for loop is where am not seeing the mechanics of the loop
+//i let the index start at 2, index will be less than the total number
+//within the code block the values in "seq" is passd through and pushed.
+//Having the it start at two requires reducing the index value of 2 to 1 and 2 to 0
+
+
+
+function fib(nums) {
+
+    let seq = [1, 1];
+    let data = [seq[0], seq[1]];
+
+    for (let i = 2; i < nums; i++) {
+        seq[i] = seq[i - 1] + seq[i - 2];
+        data.push(seq[i]);
+    }
+    return data;
+}
```

Comment:
Your approach looks great. It seems to me that `seq` and `data` are holding the same data.